examples/twitter.amp.html

```diff
@@ -20,30 +20,45 @@
       layout="responsive"
       data-tweetid="638793490521001985">
       <blockquote placeholder class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I only needed to change some CSS. <a href="http://t.co/LvjLbjgY9F">pic.twitter.com/LvjLbjgY9F</a></p>&mdash; Malte Ubl (@cramforce) <a href="https://twitter.com/cramforce/status/638793490521001985">September 1, 2015</a></blockquote>
+      <div fallback>
+        Something happened..
```

Comment:
We just don't want to change the behavior. As long as **without a fallback** the placeholder is still shown, that is fine.